### PR TITLE
fix/8129: remove rounding the caused issue with UI

### LINKED ISSUE #8129: Filter by Price: slider position is out of sync with the thumb
## Describe the bug
There's a UI glitch in Filter by Price. When moving the thumb, the slider doesn't exactly follow the thumb position. It matches the position every couple of steps (see the screenshots) and is out of sync in between

## To reproduce
Prerequisites:
- Use a wide screen (min. 1500px wide). Issue occurs on a smaller screens as well, but the position difference is smaller than the  thumb radius, hence the glitch is "covered" by the thumb.

1. Create a new page
2. Add a Filter by Price block
3. Save and see go to frontend
4. Move the thumbs slowly

## Expected behavior

Slider is following the thumbs

## Screenshots/videos

- 1500px wide page
Screen Shot 2023-01-09 at 17 37 39 PM

- 2300px wide page (on the video, you can see that every couple of steps slider and thumb go in sync and then the position difference increases until the next step when they sync again)
https://user-images.githubusercontent.com/20098064/211360425-5f9126ce-365c-47f1-8184-862bb55308b8.mov

### WordPress

-   WordPress version: 6.1.1
-   Gutenberg version (if installed): 
-   WooCommerce version: 7.2.0
-   WooCommerce Blocks version: 9.2.0

### Desktop (please complete the following information):

-   OS: macOS
-   Browser: Chrom

### PR DESCRIPTION
**Pull request by Sidsector9**
Fixes #8129 

Thie PR removes the rounding of `low` and `high` values that caused issue with the range slider's UI.

#### Accessibility

- [ ] I've tested using only a keyboard (no mouse)
- [ ] I've tested using a screen reader
- [ ] All animations respect [`prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)
- [ ] All text has [at least a 4.5 color contrast with its background](https://webaim.org/resources/contrastchecker/)

#### Other Checks

- [ ] This PR adds/removes a feature flag & I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
- [ ] This PR adds/removes an experimental interfaces and I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
- [ ] I tagged two reviewers because this PR makes queries to the database or I think it might have some security impact.

#### Before

Screenshot 2023-04-19 at 7 15 18 PM

#### After

Screenshot 2023-04-19 at 7 15 54 PM

### Testing

1. Use a wide screen (min. 1500px wide). Issue occurs on a smaller screens as well, but the position difference is smaller than the thumb radius, hence the glitch is "covered" by the thumb.
2. Create a new page
3. Add a Filter by Price block
4. Save and see go to frontend
5. Move the thumbs slowly
6. Expected: Thumb is in synced position with the slider

Please see the issue for more details.

#### Automated Tests
* [ ] Changes in this PR are covered by Automated Tests.
  * [ ] Unit tests
  * [ ] E2E tests

* [ ] Do not include in the Testing Notes 

### WooCommerce Visibility

* [ ] WooCommerce Core
* [ ] Feature plugin
* [ ] Experimental

### Changelog

> Add suggested changelog entry here.

### CODE REVIEW

File: assets/js/base/components/price-slider/index.tsx [hunk 1]
```diff
@@ -147,18 +147,13 @@ const PriceSlider = ( {
...
-			) - 0.5;
-		const high =
-			Math.round(
-				100 *
-					( ( maxPrice - minConstraint ) /
-						( maxConstraint - minConstraint ) )
-			) + 0.5;
+		const low = 100 * (
+			( minPrice - minConstraint ) / ( maxConstraint - minConstraint )
+		);
```
Comment (nefeline) [inline assets/js/base/components/price-slider/index.tsx]:
```suggestion
		const low =
			100 *
			( ( minPrice - minConstraint ) /
				( maxConstraint - minConstraint ) );
```

File: assets/js/base/components/price-slider/index.tsx [hunk 2]
```diff
@@ -147,18 +147,13 @@ const PriceSlider = ( {
...
+
+		const high = 100 * (
+			( maxPrice - minConstraint ) / ( maxConstraint - minConstraint )
+		);
```
Comment (nefeline) [inline assets/js/base/components/price-slider/index.tsx]:
```suggestion
		const high =
			100 *
			( ( maxPrice - minConstraint ) /
				( maxConstraint - minConstraint ) );
```

**Review by nefeline — CHANGES_REQUESTED** [review]
Hi there @Sidsector9 !

Thank you so much for this contribution! I ran the tests and confirmed the problem is now solved.

Before:

Screenshot 2023-04-19 at 13 12 45

After: 

Screenshot 2023-04-19 at 13 13 29

My only recommendation is to fix the JS lint errors. After that, we should be good to go! 🚢 

As a side note, it is possible to run the lint tests locally as well via `npm run lint:js`

**Review by nielslange — COMMENTED** [review]
Hallo @Sidsector9 👋 Thank you so much for your contribution. Besides the changes that @nefeline suggested, could you add testing steps to the PR description? This will help a lot in the future when this PR will get merged into the WooCommerce core. 😀

**Comment by Sidsector9** [comment]
@nielslange @nefeline both feedbacks are addressed.

**Review by nefeline — APPROVED** [review]
Thanks for addressing all feedback @Sidsector9 !

Even though the test instructions were listed on the issue, it is always a good practice to add them to the description of PRs to expedite work for the reviewers: thanks, @nielslange, for this excellent suggestion!

All tests are now passing successfully, so I'm going ahead and merging this PR.

Thanks again for contributing to Woo Blocks 🎉 !

**Pull request merged**